main.py

```diff
@@ -343,6 +343,29 @@ def multi_search(version=DEFAULT_API_VERSION):
     return data
 
 
+@app.route('/api/<version>/tv/multi-search/',
+           endpoint='multi-search-tv')
+def multi_search_tv(version=DEFAULT_API_VERSION):
+    query = request.args.get('q', '')
+    num_results = 0 if query == 'empty' else 42
+
+    kw = {
+        # The doc_type parameter in the API is singular even though it can
+        # contain multiple document types, separated by a comma. It defaults to
+        # webapp,website if absent.
+        'doc_types': request.args.get('doc_type', 'webapp,website').split(',')
+    }
+
+    if query.startswith('num-previews-'):
```

Comment:
This is probably unnecessary since the TV frontend does not feature previews of any kind.